relayer/README.md

```diff
@@ -50,7 +50,7 @@ Example Configuration:
 ```toml
 [ethereum]
 endpoint = "ws://localhost:8545/"
-descendants-until-final = 0
+descendants-until-final = 35
```

Comment:
Just an FYI, a value of `35` caused the relayer to break during E2E tests (segfaults, etc).

Reply:
Added https://github.com/Snowfork/polkadot-ethereum/issues/256 to track

Reply:
I'm guessing I introduced an underflow error at https://github.com/Snowfork/polkadot-ethereum/blob/bd53ee173bd04b55ccdf179464e3e0d56b1d3e82/relayer/chain/ethereum/listener.go#L114 when I switched to bulk message sends. I'll fix, although it shouldn't be a problem if the e2e config keeps using 0 (which is what it should be using?)

I'm using 0 in e2e config so that blocks are immediately finalized. So messages are immediately forwarded. It makes e2e test runs *much* faster. But I'm weary of putting 0 in a readme config in case it makes its way into production - 0 is only ok when forks aren't possible.